### PR TITLE
Added next/previous buttons to pages

### PR DESCRIPTION
**Pull request by crotger**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -99,6 +99,7 @@ def get(cls, pid):
         json_uri='https://%s/api/pub/items/%s/?q=*&fl=*' % (app_settings.BDR_SERVER, pid)
         resp = requests.get(json_uri)
         if not resp.ok:
+             return {"id": json_uri, "title":resp.status_code}
```
Comment (bcail) [inline models.py]: can this line be removed?

Comment (crotger) [inline models.py]: Yes, that was a testing line I forgot to remove

File: views.py [hunk 2]
```diff
@@ -154,6 +154,39 @@ def page_detail(request, page_id, book_id=None):
         curr_annot = get_annotation_detail(annotation)
         context['annotations'].append(curr_annot)
 
+    # Previous/next page queries
+    pagenum = int(page_json['rel_has_pagination_ssim'][0])
+    base_url = "https://%s/api/pub/search/?q=ir_collection_id:621+AND+rel_is_part_of_ssim:\"%s\"+AND+%s+AND+display:BDR_PUBLIC&fl=pid,rel_has_pagination_ssim,rel_is_part_of_ssim"
+    full_url = base_url % (BDR_SERVER, book_pid, "(rel_has_pagination_ssim:%d+OR+rel_has_pagination_ssim:%d)" % (pagenum - 1, pagenum + 1))
+
+    r = requests.get(full_url, timeout=60)
```
Comment (bcail) [inline views.py]: It looks like there's a book_json variable earlier in this function. That should contain a 'relations' section, which has a 'hasPart' section (eg. see https://repository.library.brown.edu/api/pub/items/bdr%3A253364/). That 'hasPart' section lists all the pages of the book (in order), so you should be able to grab the prev/next pids from that list. That way, we'd be able to eliminate one call to the api, which should speed up the page load a bit.

Comment (crotger) [inline views.py]: Ah okay--didn't realize the page was doing a book query as well. I'll close this request and rework things.

**Pull request closed without merging**